src/DynamoCore/Graph/Nodes/NodeInputData.cs

```diff
@@ -104,6 +136,18 @@ public class NodeInputData
             { typeof(Int64),NodeInputTypes.numberInput},
             {typeof(float),NodeInputTypes.numberInput},
         };
+        private static Dictionary<Type, InputTypes> dotNetTypeToInputType = new Dictionary<Type, InputTypes>
```

Comment:
can't we reuse the old dictionary `dotNetTypeToNodeInputType` ?

Reply:
How? It returns types from the old enum? What are you thinking?

Reply:
and it's still used to set `Type` to the correct values since we serialize both of them for the time being.

Reply:
Is it a requirement that they stay the same until only one remains ?

Reply:
`Is it a requirement that they stay the same until only one remains ?` I think that is best from an API point of view, we don't know who is using it external to adsk - it's a public property in what we called a "schema" and in. the JSON anyone could be parsing.